src/index.css

```diff
@@ -1,13 +1,27 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/199101528-d507a03f-4b28-4513-a48f-4c7671f6dbed.PNG)

- [ ] Kindly make your readme descriptive. It is the documentation of your project. So please take it seriously 😊 You can make your readme descriptive using this [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍